src/components/GettingStartedTour/TourPopup.jsx

```diff
@@ -173,30 +184,30 @@ function Goal({ startSubscriptions, setIsFulfilled, hasGoals, stopSubscriptions,
     setIsFulfilled(areAllConditionsFulfilled);
   },[areAllConditionsFulfilled]);
 
-  // Create animated click - it requires the component to be render fairly often
-  const tutorial = useContextRefs();
-  let lastKey = null;
-  const newElement = document.createElement('div');
-  const animationDiv = React.useRef(newElement);
-  if(content?.key && !areAllConditionsFulfilled) {
-    // Find last ref that is not null
-    const keyCopy = [...content.key].reverse();
-    lastKey = keyCopy.find(key => Boolean(tutorial.current[key]));
-  }
-  
-  React.useEffect(() => {
-    newElement.className = styles.clickEffect;
-    tutorial.current[lastKey]?.appendChild(animationDiv.current);
-
-    return () => {
-      try {
-        tutorial.current[lastKey]?.removeChild(animationDiv.current);
-      }
-      catch(e) {
-        console.error("Error: " + e);
+  // If animation should be displayed
+  // Update the animation click so it is placed where it should be
+  if(animationDiv.current) {
```

Comment:
nitpicky ,but please add a space before the first bracket in all the `if`s

Maybe we should start looking at setting up those linting rules?

Reply:
Sounds good!